src/core/include/units/bits/basic_concepts.h

```diff
@@ -38,19 +38,22 @@
 namespace units {
 
 // PrefixFamily
+UNITS_EXPORT
 struct prefix_family;
 
 /**
  * @brief A concept matching a prefix family
  *
  * Satisfied by all types derived from `prefix_family`
  */
+UNITS_EXPORT
 template<typename T>
 concept PrefixFamily = std::derived_from<T, prefix_family>;
 
 // Prefix
 namespace detail {
 
+UNITS_LLVM_EXPORT
```

Comment:
Yes, all added macros prefixed with `UNITS_LLVM_` are about LLVM reachability bugs, and should go away when they're fixed and support for versions of Clang that require them is dropped.

Reply:
Turns out it was because I was using Clang modules, which is different from Clang's implementation of C++ modules. The former was documented and the latter not, so I thought they were the same. Now you can find the latter's here: https://clang.llvm.org/docs/StandardCPlusPlusModules.html.